app/model/exchange/passmarksettings/PassMarkSettings.scala

```diff
@@ -52,3 +52,16 @@ object Phase2PassMarkSettings {
   implicit val phase2PassMarkSettingsFormat = Json.format[Phase2PassMarkSettings]
   implicit val phase2PassMarkSettingsHandler = Macros.handler[Phase2PassMarkSettings]
 }
+
+case class Phase3PassMarkSettings(
+  schemes: List[Phase3PassMark],
```

Comment:
The way I see Phase3PassMarkSettings is that it is a list of passmarks (which is schemes field) and then some metadata. So schemes field is actually the passmarkSettings without metadata. In that sense calling them schemes is a bit misleading, I would exactly what it is exactly is passmarkSettingsPerScheme or passmarksPerScheme